packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -171,33 +171,6 @@ type Props = AriaProps &
         dropdownId?: string;
     }>;
 
-type State = Readonly<{
-    /**
-     * Whether or not the dropdown is open.
-     */
-    open: boolean;
-    /**
-     * The text input to filter the items by their label. Defaults to an empty
-     * string.
-     */
-    searchText: string;
-    /**
-     * The selected values that are set when the dropdown is opened. We use
-     * this to move the selected items to the top when the dropdown is
-     * re-opened.
-     */
-    lastSelectedValues: Array<string>;
-    /**
-     * The object containing the custom labels used inside this component.
-     */
-    labels: Labels;
-    /**
-     * The DOM reference to the opener element. This is mainly used to set focus
-     * to this element, and also to pass the reference to Popper.js.
-     */
-    openerElement?: HTMLElement;
-}>;
```

Comment:
`open`, `searchText`, `lastSelectedValues` and `openerElement` have `useState` equivalents
https://github.com/Khan/wonder-blocks/blob/e02ada1e89fd30c08b34fabba5658e2547541640/packages/wonder-blocks-dropdown/src/components/multi-select.tsx#L223-L238

`labels` doesn't need to be in state and combines the default labels and prop labels: 
https://github.com/Khan/wonder-blocks/blob/e02ada1e89fd30c08b34fabba5658e2547541640/packages/wonder-blocks-dropdown/src/components/multi-select.tsx#L220-L221